### ISSUE TITLE
Attack Problem

### ISSUE THREAD

**Issue by JennaDiamonds** [mcmonkey4eva/Sentinel#189]
Not sure if this is normal or not
After a sentinel attacks a player, the sentinel turns 180 degrees and takes about a step away from the player, then turns back around and comes up for another attack. Can this behavior be fixed/disabled?

I know i keep posting a lot in here. But atm I'm working quite heavily on enemies in my server :)

**Comment by mcmonkey4eva** [comment]
Are you running latest dev builds? I remember that happening on earlier builds, but the AI's been cleaned a bit since then...

**Comment by JennaDiamonds** [comment]
using citizens 2 http://ci.citizensnpcs.co/job/Citizens2/lastBuild/
using sentinel http://ci.citizensnpcs.co/job/Sentinel/

Far as i know these are the latest builds :(

**Comment by JennaDiamonds** [comment]
**UPDATE** The previously mentioned "attacking in circles" only happens until the sentinel is attacked for the first time after spawning. After being attacked, his pathfinding works just fine.

**Comment by JennaDiamonds** [comment]
**UPDATE** Strangely, and this doesn't seem to be mentioned anywhere in citizens' documentation.. in the citizens Config.yml- the setting "Use-new-finder" is by default set to false. I set it to true.. out of desperation since there is no explanation for the setting. And now my citizens not only are attacking a bit more properly (still circles until they're hit once), but they also navigate stairs about 10x better. Before they were just getting stuck on them all the time. Totally random that I found this option.. But happy I spent the time troubleshooting haha
